internal/k8s/builder/gateway.go

```diff
@@ -249,8 +252,13 @@ func (b *GatewayDeploymentBuilder) podSpec() corev1.PodSpec {
 						},
 					},
 				},
+				{
+					Name:  api.HTTPCAFile,
+					Value: consulCALocalFile,
+				},
 			},
-			Command: b.execCommand(),
+			Command: []string{"/bootstrap/consul-api-gateway", "exec"},
+			Args:    b.execArgs(),
```

Comment:
Instead of running the shell with one giant argument that was our startup script, we're now running `consul-api-gateway exec` with many small arguments/flags